imagefiles/install-cmake-binary.sh

```diff
@@ -35,11 +35,26 @@ fi
 
 cd /usr/src
 
-CMAKE_ROOT=cmake-${CMAKE_VERSION}-Centos5-${ARCH}
-url=https://github.com/dockbuild/CMake/releases/download/v${CMAKE_VERSION}/${CMAKE_ROOT}.tar.gz
+#Switch to newer version of CMake
+#USER=dockbuild #Original user, change it when changed it when there will be a new update of CMake (3.17.1 on June 27, 2021)
+
+USER=bensuperpc
```

Comment:
Hmm not sure how I feel about giving control to a user repo. I think @jcfr and @thewtex are involved in dockbuild, maybe they could update CMake there instead...

Reply:
This is a better idea, I didn't know if the account was still active or not, I made a PR for CMake with the changes (update openssl 1.1.1j, libuv 1.41, centos 7 ect ...) a few days ago.

Reply:
> Hmm not sure how I feel about giving control to a user repo. I think @jcfr and @thewtex are involved in dockbuild, maybe they could update CMake there instead...

This is a better idea, I didn't know if the account was still active or not, I made a PR for CMake with the changes (update openssl 1.1.1j, libuv 1.41, centos 7 ect ...) a few days ago.